source/style/a11y.css

```diff
@@ -0,0 +1,12 @@
+/* Plugin-generated div that display the alternative text of an image */
+div.refined-twitter_alt {
```

Comment:
Drop the `div`

Reply:
`.refined-twitter_alt` => `.refined-twitter_image-alt` would be more descriptive

Reply:
👍